plugins/available/node.plugin.bash

```diff
@@ -1,13 +1,15 @@
+# shellcheck shell=bash
 cite about-plugin
 about-plugin 'Node.js helper functions'
 
+# Check that we have npm
+_command_exists npm || return
+
 # Ensure local modules are preferred in PATH
 pathmunge "./node_modules/.bin" "after"
 
-# Check that we have npm
-out=$(command -v npm 2>&1) || return
-
 # If not using nodenv, ensure global modules are in PATH
-if [[ ! $out == *"nodenv/shims"* ]] ; then
+if [[ ! "$(type -p npm)" == *"nodenv/shims"* ]]
```

Comment:
This could probably use [`_binary_exists`](https://github.com/Bash-it/bash-it/blob/master/lib/helpers.bash#L29)